### ISSUE TITLE
Unable detect audit alerts Ubuntu 16.04 LTS wazuh-agent

### ISSUE THREAD

**Issue by c0r3dump3d** [wazuh/wazuh#12897]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.2.6-1  | Agent | deb https://packages.wazuh.com/4.x/apt/ stable main | apt install wazuh-agent| Ubuntu 16.04 LTS |

I try to detect audit.log  alert, specifically PAM login. I see these messages in audit.log 

```
type=CRED_ACQ msg=audit(1648544662.972:3932865): pid=53806 uid=0 auid=1003 ses=552979 msg='op=PAM:setcred acct="admin" exe="/usr/sbin/sshd" hostname=10.100.10.236 addr=10.100.10.236 terminal=ssh res=success'
type=USER_LOGIN msg=audit(1648544663.124:3932866): pid=53763 uid=0 auid=1003 ses=552979 msg='op=login id=1003 exe="/usr/sbin/sshd" hostname=10.100.10.236 addr=10.100.10.236 terminal=/dev/pts/3 res=success' 
```
But I am not able to make them appear in Wazuh

I have all the decoders and ruleset installed in /var/ossec/ruleset
```
/var/ossec/ruleset# ls -lah
total 52K
drwxr-x---  8 root ossec 4.0K Mar 29 09:11 .
drwxr-x--- 15 root ossec 4.0K Mar 29 09:24 ..
drwxr-xr-x  2 root ossec  12K Mar 29 09:10 decoders
drwxr-xr-x  3 root ossec 4.0K Mar 29 09:10 lists
drwxr-xr-x  2 root ossec 4.0K Mar 29 09:10 mitre
-rw-r--r--  1 root ossec 2.8K Mar 29 09:10 README.md
drwxr-xr-x  3 root ossec 4.0K Mar 29 09:10 rootcheck
drwxr-xr-x  2 root ossec  12K Mar 29 09:10 rules
drwxr-x--- 16 root ossec 4.0K Mar 29 09:24 sca
```
And in ossec.conf:
```
<localfile>
    <log_format>audit</log_format>
    <location>/var/log/audit/audit.log</location>
</localfile>

```

I have other system (Debian 10, CentOS 7) with the same configuration and these alerts works very well. 

I have activated agent.debug = 2, but no errors appear.

**Comment by jotacarma90** [comment]
Hello @c0r3dump3d 
In order to debug this problem, let's take several steps:

- To see the debug correctly, you should add the line `logcollector.debug=2`.
- You can check that the audit rules are set correctly with the command `auditctl -l`.
- You can check the step by step process to configure everything in this blogpost:
https://wazuh.com/blog/monitoring-root-actions-on-linux-using-auditd-and-wazuh/

- Related documentation in case it can help you here:
https://documentation.wazuh.com/current/proof-of-concept-guide/audit-commands-run-by-user.html

- Is it in Kibana where you are missing the alerts? Could you check the `/var/ossec/logs/alerts/alerts.log` file of the manager? (so we can focus more on Kibana)

Any clue would be helpful, thank you very much!!!
Regards

**Comment by jotacarma90** [comment]
Hi @c0r3dump3d 

`/var/ossec/logs/alerts/alerts.log` is a file on the manager side only, can you check it?
The flow of events in Wazuh follows this scheme:
- Wazuh-agent generates the event and sends it to wazuh-manager.
- The manager processes and filters it, if it exceeds the minimum alert level, among other things, it generates an alert (the alerts can be found in the mentioned file).
- The manager sends the alert to Kibana to be displayed in the interface.

So checking if the alert is in the alerts.log file can make it easier to know where a problem is occurring.

**Comment by c0r3dump3d** [comment]
> /var/ossec/logs/alerts/alerts.log

Hi, I have checked /var/ossec/logs/alerts/alerts.log and I can not see any alert from the Ubuntu system, for example in a CentOS 7 when I have made a ssh login I see in the manager:

`Mar 31 08:08:59 test1 sshd[14304]: pam_unix(sshd:session): session opened for user admin by (uid=0)`

but not in Ubuntu.

**Comment by c0r3dump3d** [comment]
On the other hand  I see that in the Ubuntu system logs from wazuh that it is sending audit messages to the server:

```
> 2022/03/31 09:21:52 wazuh-agentd[24038] notify.c:130 at run_notify(): DEBUG: Sending agent notification.
> 2022/03/31 09:21:57 wazuh-logcollector[24073] read_audit.c:149 at read_audit(): DEBUG: Read 2 lines from /var/log/audit/audit.log
> 2022/03/31 09:21:59 wazuh-logcollector[24073] read_audit.c:149 at read_audit(): DEBUG: Read 2 lines from /var/log/audit/audit.log
> 2022/03/31 09:22:02 wazuh-agentd[24038] notify.c:130 at run_notify(): DEBUG: Sending agent notification.
> 2022/03/31 09:22:03 wazuh-logcollector[24073] read_audit.c:149 at read_audit(): DEBUG: Read 18 lines from /var/log/audit/audit.log
> 2022/03/31 09:22:12 wazuh-agentd[24038] notify.c:130 at run_notify(): DEBUG: Sending agent notification.
> 2022/03/31 09:22:22 wazuh-agentd[24038] notify.c:130 at run_notify(): DEBUG: Sending agent notification.
> 2022/03/31 09:22:32 wazuh-agentd[24038] notify.c:130 at run_notify(): DEBUG: Sending agent notification.

```
But I don't see any message in the `alerts.log` from the audit Ubuntu system.

**Comment by jotacarma90** [comment]
I have been checking with the log testing tool that we have in Wazuh, `/var/ossec/bin/wazuh-logtest.` You can check it yourself, you just need to paste the whole log.

I see that those audit logs that you have sent, get a level 0 of the rules, so they do not generate an alert. Could you send me examples of the alerts that you can see in centos to better understand the problem?

**Comment by c0r3dump3d** [comment]
Sorry!! I just discovered the problem, I was confused. In CentOS the alert doesn't come from audit but the log file `/var/log/secure`, in Ubuntu comes from `/var/log/auth.log`, I have changed `ossec.conf` to include this log file and now it's working. 

Sorry for this stupid question. I take this opportunity to congratulate you because Wazuh is a fantastic tool, and thank you very much for the help.
